service/models.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from flask import Flask
 from mongoengine import Document, StringField, ListField, IntField, DateTimeField, connect
-from mongoengine.queryset.visitor import Q
```

Comment:
I think we should keep the Q function because the following action operation uses Q

Reply:
> I think we should keep the Q function because the following action operation uses Q

Thanks @silemie ! I mistakenly removed it when resolving merge conflicts. 
Have been restored.